test/runtests.jl

```diff
@@ -26,7 +26,7 @@ using LinearAlgebra
 pms_path = joinpath(dirname(pathof(PowerModels)), "..")
 pmd_path = joinpath(dirname(pathof(PowerModelsDistribution)), "..")
 
-ipopt_solver = with_optimizer(Ipopt.Optimizer, tol=1e-6, print_level=0)
+ipopt_solver = with_optimizer(Ipopt.Optimizer, tol=1e-7, print_level=0)
```

Comment:
Reason for the change in tolerance?

Reply:
This solved some issues I was having. Though after investigsting it more, it turned out some test from your last commit got overwritten I believe, causing the issue. 

In any case, this is not needed anymore, I removed it.